s3_to_redshift/s3_to_redshift.py

```diff
@@ -329,56 +331,80 @@ def report(data):
 
     # This is not an apache access log
     if 'access_log_parse' not in data:
-        csv_string = body.read()
+        if obj['ResponseMetadata']['HTTPHeaders']['content-type'].endswith('sheet'):
```

Comment:
Instead of booleans csv and xlsx, I suggest we set a variable like fileformat and populate it with the format. 

Also, I think we want this configured by the JSON, rather than parsing an HTTP Header for content-type.

I'm not sure that "endswith('sheet')" is a robust long term solution to file types.

Reply:
Most of the logic that was intended to make the script content aware is going to be scraped. We already hold tight control on what the script expects and shouldn't steer away from that. A content-type flag in the config file is a better route, for sure.